onboard-blink/cpp/README.md

```diff
@@ -1,872 +1,31 @@
+# Onboard Blink 
 
 
+## Introduction
+This is a simple sample you could use for a quick test.
 
+## What it is
+A blink sample, it turns the on-board LED on for one second, then off for one second, repeatedly.
 
+## Hardware requirements
 
+A board with an accesible GPIO pin (connected to an LED).
 
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
-  <link rel="dns-prefetch" href="https://assets-cdn.github.com">
-  <link rel="dns-prefetch" href="https://avatars0.githubusercontent.com">
-  <link rel="dns-prefetch" href="https://avatars1.githubusercontent.com">
-  <link rel="dns-prefetch" href="https://avatars2.githubusercontent.com">
-  <link rel="dns-prefetch" href="https://avatars3.githubusercontent.com">
-  <link rel="dns-prefetch" href="https://github-cloud.s3.amazonaws.com">
-  <link rel="dns-prefetch" href="https://user-images.githubusercontent.com/">
+## Supported boards
 
+This sample has been tested on
+- MinnowBoard Turbot 2.0
+- Up2 board
```

Comment:
Should this have some trademark or registered mark or asterisk? because they are registered names right?